### ISSUE TITLE
.opus audio not working on Ankimobile despite other .opus audio working on iOS 18

### ISSUE THREAD

**Issue by andy212800** [themoeway/local-audio-yomichan#35]
On iOS 18, I have .opus audio from ShareX working on Ankimobile, but neither the .ogg audio when added from Yomitan, nor the .opus taken directly from the audio collection play on Ankimobile.

**Comment by jamesmaa** [comment]
> However, Opus is NOT compatible with AnkiMobile (iOS), Android 4, and AnkiWeb. If you use any of these, please use the 2nd option (MP3 audio) below.

From the [README](https://github.com/themoeway/local-audio-yomichan)

**Comment by andy212800** [comment]
Yes, I know that, but as I stated, I got .opus audio working from another source on iOS 18. So, I was thinking maybe the opus files could be re-encoded or something so that they work. Perhaps the version of the .opus file matters, since the ShareX .opus files were encoded using recent versions of ffmpeg.
